src/server/resources/queries/LATEST_TEST_USER.sql

```diff
@@ -1,3 +1,3 @@
 select ifnull(MAX(person.username), 'TEST0000') as username
```

Comment:
This should probably read for completeness

select ifnull(MAX(person.username), 'VARC0000') as username